euclidean_gcd/euclidean_gcd.js

```diff
@@ -0,0 +1,25 @@
+function euclidean_gcd_recursive(first, second) {
```

Comment:
No, the naming convention in java and javascript is `camelCase`, while c, cpp and python has underscores.